.github/workflows/ci.yaml

```diff
@@ -27,6 +27,9 @@ jobs:
     env:
       POLYGONSCAN_API_KEY: ${{ secrets.POLYGONSCAN_API_KEY }}
       WEB3_INFURA_PROJECT_ID: ${{ secrets.WEB3_INFURA_PROJECT_ID }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.DIGITALOCEAN_S3_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.DIGITALOCEAN_S3_SECRET_ACCESS_KEY }}
+      PYTHONUNBUFFERED: 1
```

Comment:
Required to see logs. Otherwise /tmp/nohup is not complete when it is read by the smoketest